Reword unpadded PATH_CHALLENGE language (#4580)
quicwg/base-drafts: draft-ietf-quic-transport.md

frame received on any network path validates the path on which the
PATH_CHALLENGE was sent.

> A PATH_CHALLENGE frame that was sent in a datagram that was not expanded to at
> least 1200 bytes validates the peer address, but it does not validate the path

Reviewer: ```suggestion
A PATH_CHALLENGE frame sent in a datagram that was not expanded to at
least 1200 bytes validates the peer address, but it does not validate the path
```

Author: Thanks, but I'll tweak further.

> A successful response to a PATH_CHALLENGE frame sent in a datagram that was not
> expanded to at least 1200 bytes validates the peer address, but it does not
> validate the path MTU. The endpoint is then able to send more than three times
> the amount of data that has been received. However, the endpoint MUST initiate
> another path validation with an expanded datagram to verify that the path
> supports required MTU.

Reviewer 2: ```suggestion
If an endpoint sends a PATH_CHALLENGE frame in a datagram that is not expanded
to at least 1200 bytes, and if the response to it validates the peer address,
that path is validated but not the path MTU. As a result, the endpoint can now
send more than three times the amount of data that has been received but it MUST
initiate another path validation with an expanded datagram to verify that the
path supports required MTU.
```
